### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD008.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#16] [General]
# https://blog.3place.cn/3place/fsd/FSD008.html

https://blog.3place.cn/3place/fsd/FSD008.html

**Comment by picklecai** [comment]
每天都像过节一样
不动脑筋，却非常开心
人人兜着小圈子盘旋舞蹈
就像戏弄自己尾巴的小猫

听上去十分可笑。可其实人的大部分时间只不过如此：你以为的正经事一点意义都没有，你以为的忙碌完全是在浪费生命。等最后一天来临时追悔也没用。

浮士德不是刚刚抱怨过这样的日子吗？怎么又领他来看同样的场景？

**Comment by picklecai** [comment]
它心慌意乱，跳个不停
可怜它跳得筋疲力尽
好像害了相思病。

中毒的表现，比喻为相思病。仔细想想，相思病，何尝不是一种中毒？还是无药可解的毒呢。

**Comment by picklecai** [comment]
他们认不出恶魔的嘴脸
即使他抓住他们的衣领。

大隐隐于市的既视感。那些逃亡二三十年的人，或者潜伏很久的间谍，都有这让人“过目即忘”的本领。
